nixos/services/mail/default.nix

```diff
@@ -200,7 +215,27 @@ in {
               inherit (role) mailHost webmailHost;
             });
           } config.fclib.network.fe.dualstack.addressesQuoted);
-        in listToAttrs (map cfgForDomain role.domains);
+        in listToAttrs ((map cfgForDomain role.domains) ++
+          [ (lib.nameValuePair fqdn
+            (lib.mkMerge [
+              {
+                addSSL = true;
+                enableACME = true;
+              }
+              (lib.mkIf (role.imprintUrl != null) {
+                globalRedirect = role.imprintUrl;
+              })
+              (lib.mkIf (role.imprintText != null) {
+                root = with pkgs; writeTextDir "index.html" role.imprintText;
+              })
+              (lib.mkIf (role.imprintUrl == null && role.imprintText == null) {
+                root = with pkgs; writeTextDir "index.html" ''
+                  <h1>Imprint missing</h1>
+                  <h2>Imprint can be configured using <tt>flyingcircus.roles.mailserver.imprintText</tt> or <tt>flyingcircus.roles.mailserver.imprintUrl</tt></h2>
```

Comment:
A link to the documentation at https://doc.flyingcircus.io/roles/fc-21.05-production/mailserver.html#how-do-i-perform-a-basic-setup would be helpful.